trajectory_msgs/JointTrajectory joint_trajectory
---
bool response